### ISSUE TITLE
Revise Text Styles in Figma's Design System 

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2601]
### Overview
 We noticed that the Titles and Headers in the design system needs to be revised because the only difference between text variations are the size and boldness of them. How can we revise the typography section of the Design System to work better together without size and boldness of fonts being the only variation options between Titles and Headers? 

### Action Items
- [x] Create a Figma page to work on this issue in and name it " Revised Text Styles_Issue 2601". Link it in the resources section of this issue
- [x] Revise the Typography  section on the new figma page. You may use recommendations from
  - #2361
  - https://github.com/hackforla/website/issues/1797#issuecomment-1001077278 - This is an example from another design system where additional weights are provided
- [ ] Present your revised text typography to the team for feedback
- [ ] Once the new text style typography is approved update the design system with them.
- [ ] Present the finished issue to the team.

Next Steps
- [ ] We need to make an issue to create and update the Text Styles in the Design System so that we can easily reuse them in the future. Then, another issue to apply the new text styles to each pages latest desktop and mobile designs (In red container).

### Resources/Instructions

- [Hack4LA Design System](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)
- [Font Weight Audit](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=18792%3A126489)
- [Revised Text Styles_Issue 2601](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/?node-id=19750%3A127092)

#### External advice

- https://www.designsystems.com/typography-guides/
- https://blog.prototypr.io/10-practical-steps-to-create-a-predictable-accessible-and-harmonious-typography-system-a-case-6c85d901bedd
- https://medium.com/eightshapes-llc/typography-in-design-systems-6ed771432f1e

**Comment by IsaacDesigns** [comment]
Progress: To make this issue more efficiently worked on I must first create an issue to connect the design system's components/styles to to Figma's latest screen designs. 
Blockers: None
Availability: n/a
ETA: This week

**Comment by IsaacDesigns** [comment]
I've rewritten this issue and will talk to @ExperimentsInHonesty about what I will do about connecting the latest Figma designs to the design system.

**Comment by ExperimentsInHonesty** [comment]
This issue originated with my discussion with Isac about why we don't have multiple weights for fonts see this for example: https://github.com/hackforla/website/issues/1797#issuecomment-1001077278

While it is necessary to refine the design system fonts, its important to have an overall plan first.

**Comment by ExperimentsInHonesty** [comment]
@nathan944 Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by nathan944** [comment]
1. Progress: So far I've been able to create the different variations for each title.  Right now, I still have to change and separate each variation by its class 
2. Blockers: None
3. Availability: 20/hours
4. ETA: I'll have it ready for review for Wednesday's meeting (6/1/22), if everything is ok during review the issue should then be ready to be added to the design system, but for now I don't have a set date for the actual completion of the project. 
5. Pictures (if necessary): 
![image](https://user-images.githubusercontent.com/76053494/170886937-26566bd3-75f8-4fc5-a14f-b9a728b1f809.png)

**Comment by nathan944** [comment]
Progress: I finished the draft of the new text style typography, which meant separating the classes and variations as discussed in the last meeting. Currently, we need to review it with the whole team to see if they approve of it and whether the developers believe they can implement it.
Blockers: None
Availability: 9 hours this week
ETA: If the team approves everything on Sunday (6/5/22) then I'll move into updating the design system and that should then be ready for review on Wednesday (6/8/22) if everything is approved it should be completed by then. 
Pictures (if necessary):
![image](https://user-images.githubusercontent.com/76053494/171518029-6af9a3cc-c427-4d77-b490-ba9923601901.png)

**Comment by nathan944** [comment]
Bonnie with the help of Phu will look further into this issue, to get developers to come up with a way to implement the potential new text style.
